SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -11,43 +11,34 @@ ServerStatusWatcher {
 	var <avgCPU, <peakCPU;
 	var <sampleRate, <actualSampleRate;
 
-	var reallyDeadCount = 0;
-	var <>bootNotifyFirst;
-
+	var reallyDeadCount = 0, bootNotifyFirst = true;
 
 	*new { |server|
 		^super.newCopyArgs(server)
 	}
 
-	quit { |watchShutDown = true|
+	quit { |onComplete, onFailure, watchShutDown = true|
```

Comment:
Maybe oncomplete and onfailure should be after the existing watchshutdown arg, for backwards compatibility of calls that don't name the argument?